### PR TITLE
Change position of browsable exact match

### LINKED ISSUE #627: Show 2 browse entries before exact match instead of 3
Update the [browsables_controller](https://github.com/pulibrary/orangelight/blob/development/app/controllers/orangelight/browsables_controller.rb#L42) to start the browse view from 2 entries before the matching entry. For some longer entries in the call number browse table, the query match appears far down on the page.

### PR DESCRIPTION
**Pull request by eliotjordan**
When browsing, the exact match record is now three from the top. Closes #627.

### CODE REVIEW

File: spec/requests/orangelight/browse_spec.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@
       get '/browse/call_numbers.json?q=microfilm'
       r = JSON.parse(response.body)
       q_normalized = StringFunctions.cn_normalize('microfilm')
-      expect(r[3]['sort']..r[4]['sort']).to cover q_normalized
+      expect(r[2]['sort']..r[3]['sort']).to cover q_normalized
```
Comment (tampakis) [inline spec/requests/orangelight/browse_spec.rb]: I think this should be r[1] and r[2] since the position has been moved up by 2.

**Comment by eliotjordan** [comment]
Having some hugely frustrating issues with tests passing in my local instance and failing on Travis. Working on a fix.  :confounded:

**Comment by eliotjordan** [comment]
Ok. That did it. Not sure why Travis tests passed previously, but I added `rake db:seed` to the config.

**Comment by tampakis** [comment]
Oh I figured out the issue - it's this: https://github.com/pulibrary/orangelight/blob/development/spec/requests/orangelight/browse_spec.rb#L5
So in the dev instance, the DB was seeded twice. Could you remove that line in this PR as well?

**Comment by tampakis** [comment]
I think by removing the second seed, you'll have to change to r[1] and r[2]

**Comment by eliotjordan** [comment]
@tampakis No change to the position is needed. It makes sense. The first microfilm document is fourth in the array r[3]. The addition of the ` Your query "microfilm" would be here.` row skews the positioning compared with browsing using an actual call number. It all looks fine to me though.

**Comment by tampakis** [comment]
Opened issue #655 about the weird ordering of the call number browse search in dev/test.

**Pull request merged**